bin/configure.py

```diff
@@ -0,0 +1,239 @@
+import json
+import os.path
+import getpass
+import os
+
+from os import path
+
+
+def is_port_in_use(port):
+    import socket
+    with socket.socket(socket.AF_INET, socket.SOCK_STREAM) as s:
+        return s.connect_ex(('localhost', port)) == 0
+
+####doc service config
+def get_docservice_port(config):
+	fetch_docservice_port_from_user(config)
+	while config["docservice_port"] < 0 or is_port_in_use(config["docservice_port"]):
+		print(" Port : "+config["docservice_port"]+" is invalid or already open. Please enter a valid port.")
+		fetch_docservice_port_from_user(config)
+
+	
+	print("docservice_port port is :" + str(config["docservice_port"]))
+
+def fetch_docservice_port_from_user(config):
+	#check valid default port and port not in use
+	with open(defaultjson_path, "r") as f:
+			data = json.load(f)	
+
+	while True:
+		docservice_port = input("Enter docservice port number (Press enter to keep default): ")
```

Comment:
Would it be more helpful to show the default value `[default]` instead of just mentioning there is one? Same for the other prompts in this file.

Reply:
This file will no longer be needed.